docs/faq.rst

```diff
@@ -181,6 +181,17 @@ Why my file is not being included in the sdist/wheel distribution?
     being set to ``True``.
     If you want to include *transient* files in your distributions, please check `setuptools docs on data files`_.
 
+.. _git-default-branch:
+
+How can I change Git's default branch when creating a new project setup with PyScaffold?
+
```

Comment:
It seems that the formatting breaks because of this line. RST seems to require the topic (in a definition list) is followed directly by the definition.

```suggestion
```

Preview of docs: https://pyscaffold--577.org.readthedocs.build/en/577/faq.html